File: src/main/java/eu/qwsome/sql/api/Appendable.java
package eu.qwsome.sql.api;

/**
 * @author Lukáš Kvídera
 */
public interface Appendable {

  /**
   * Appends this entity to predefined builder.
   *
   * @param builder to be filled with this entity
   */
  void appendTo(StringBuilder builder);
}

File: src/main/kotlin/eu/qwsome/sql/api/Appendable.kt
/*
 * @author Lukáš Kvídera
 */
package eu.qwsome.sql.api

interface Appendable {
    /**
     * Appends this entity to predefined builder.
     *
     * @param builder to be filled with this entity
     */
    fun appendTo(builder: StringBuilder)
}